<?xml version="1.0" encoding="UTF-8" ?>
<testsuites>
 <testsuite name="angularjs homepage" timestamp="2017-07-22T13:04:43" hostname="localhost" time="176.452" errors="0" tests="1" skipped="0" disabled="0" failures="2">
  <testcase classname="angularjs homepage" name="should greet the named user" time="60.053">
   <failure type="exception" message="Error: Timeout - Async callback was not invoked within timeout specified by jasmine.DEFAULT_TIMEOUT_INTERVAL."><![CDATA[Error: Timeout - Async callback was not invoked within timeout specified by jasmine.DEFAULT_TIMEOUT_INTERVAL.
    at Timeout._onTimeout (C:\HATG\ProtractorWorkspace\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:3938:23)
    at ontimeout (timers.js:365:14)
    at tryOnTimeout (timers.js:237:5)
    at Timer.listOnTimeout (timers.js:207:5)]]>
   </failure>
   <failure type="exception" message="Error: Timeout - Async callback was not invoked within timeout specified by jasmine.DEFAULT_TIMEOUT_INTERVAL."><![CDATA[Error: Timeout - Async callback was not invoked within timeout specified by jasmine.DEFAULT_TIMEOUT_INTERVAL.
    at Timeout._onTimeout (C:\HATG\ProtractorWorkspace\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:3938:23)
    at ontimeout (timers.js:365:14)
    at tryOnTimeout (timers.js:237:5)
    at Timer.listOnTimeout (timers.js:207:5)]]>
   </failure>
  </testcase>
 </testsuite>
 <testsuite name="angularjs homepage.todo list" timestamp="2017-07-22T13:05:43" hostname="localhost" time="116.394" errors="0" tests="2" skipped="0" disabled="0" failures="7">
  <testcase classname="angularjs homepage.todo list" name="should list todos" time="30.025">
   <failure type="exception" message="Error: Timeout - Async callback was not invoked within timeout specified by jasmine.DEFAULT_TIMEOUT_INTERVAL."><![CDATA[Error: Timeout - Async callback was not invoked within timeout specified by jasmine.DEFAULT_TIMEOUT_INTERVAL.
    at Timeout._onTimeout (C:\HATG\ProtractorWorkspace\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:3938:23)
    at ontimeout (timers.js:365:14)
    at tryOnTimeout (timers.js:237:5)
    at Timer.listOnTimeout (timers.js:207:5)]]>
   </failure>
   <failure type="exception" message="Failed: Cannot read property &apos;get&apos; of undefined"><![CDATA[TypeError: Cannot read property 'get' of undefined
    at Object.<anonymous> (C:\HATG\protractor_07222017\test\example_spec.js:22:22)
    at C:\HATG\ProtractorWorkspace\node_modules\jasminewd2\index.js:112:25
    at new ManagedPromise (C:\HATG\ProtractorWorkspace\node_modules\selenium-webdriver\lib\promise.js:1067:7)
    at ControlFlow.promise (C:\HATG\ProtractorWorkspace\node_modules\selenium-webdriver\lib\promise.js:2396:12)
    at schedulerExecute (C:\HATG\ProtractorWorkspace\node_modules\jasminewd2\index.js:95:18)
    at TaskQueue.execute_ (C:\HATG\ProtractorWorkspace\node_modules\selenium-webdriver\lib\promise.js:2970:14)
    at TaskQueue.executeNext_ (C:\HATG\ProtractorWorkspace\node_modules\selenium-webdriver\lib\promise.js:2953:27)
    at asyncRun (C:\HATG\ProtractorWorkspace\node_modules\selenium-webdriver\lib\promise.js:2860:25)
    at C:\HATG\ProtractorWorkspace\node_modules\selenium-webdriver\lib\promise.js:676:7
From: Task: Run it("should list todos") in control flow
    at Object.<anonymous> (C:\HATG\ProtractorWorkspace\node_modules\jasminewd2\index.js:94:19)
    at attemptAsync (C:\HATG\ProtractorWorkspace\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:3945:24)
    at QueueRunner.run (C:\HATG\ProtractorWorkspace\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:3884:9)
    at C:\HATG\ProtractorWorkspace\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:3924:18
    at C:\HATG\ProtractorWorkspace\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:3848:9
    at Timeout.<anonymous> (C:\HATG\ProtractorWorkspace\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:3940:11)
    at ontimeout (timers.js:365:14)
    at tryOnTimeout (timers.js:237:5)
    at Timer.listOnTimeout (timers.js:207:5)
From asynchronous test: 
Error
    at Suite.<anonymous> (C:\HATG\protractor_07222017\test\example_spec.js:21:5)
    at addSpecsToSuite (C:\HATG\ProtractorWorkspace\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:994:25)
    at Env.describe (C:\HATG\ProtractorWorkspace\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:961:7)
    at describe (C:\HATG\ProtractorWorkspace\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4031:18)
    at Suite.<anonymous> (C:\HATG\protractor_07222017\test\example_spec.js:12:3)
    at addSpecsToSuite (C:\HATG\ProtractorWorkspace\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:994:25)
    at Env.describe (C:\HATG\ProtractorWorkspace\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:961:7)
    at describe (C:\HATG\ProtractorWorkspace\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4031:18)
    at Object.<anonymous> (C:\HATG\protractor_07222017\test\example_spec.js:1:1)]]>
   </failure>
  </testcase>
  <testcase classname="angularjs homepage.todo list" name="should add a todo" time="86.361">
   <failure type="exception" message="Error: Timeout - Async callback was not invoked within timeout specified by jasmine.DEFAULT_TIMEOUT_INTERVAL."><![CDATA[Error: Timeout - Async callback was not invoked within timeout specified by jasmine.DEFAULT_TIMEOUT_INTERVAL.
    at Timeout._onTimeout (C:\HATG\ProtractorWorkspace\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:3938:23)
    at ontimeout (timers.js:365:14)
    at tryOnTimeout (timers.js:237:5)
    at Timer.listOnTimeout (timers.js:207:5)]]>
   </failure>
   <failure type="exception" message="Error: Timeout - Async callback was not invoked within timeout specified by jasmine.DEFAULT_TIMEOUT_INTERVAL."><![CDATA[Error: Timeout - Async callback was not invoked within timeout specified by jasmine.DEFAULT_TIMEOUT_INTERVAL.
    at Timeout._onTimeout (C:\HATG\ProtractorWorkspace\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:3938:23)
    at ontimeout (timers.js:365:14)
    at tryOnTimeout (timers.js:237:5)
    at Timer.listOnTimeout (timers.js:207:5)]]>
   </failure>
   <failure type="exception" message="Failed: Cannot read property &apos;count&apos; of undefined"><![CDATA[TypeError: Cannot read property 'count' of undefined
    at Object.<anonymous> (C:\HATG\protractor_07222017\test\example_spec.js:33:22)
    at C:\HATG\ProtractorWorkspace\node_modules\jasminewd2\index.js:112:25
    at new ManagedPromise (C:\HATG\ProtractorWorkspace\node_modules\selenium-webdriver\lib\promise.js:1067:7)
    at ControlFlow.promise (C:\HATG\ProtractorWorkspace\node_modules\selenium-webdriver\lib\promise.js:2396:12)
    at schedulerExecute (C:\HATG\ProtractorWorkspace\node_modules\jasminewd2\index.js:95:18)
    at TaskQueue.execute_ (C:\HATG\ProtractorWorkspace\node_modules\selenium-webdriver\lib\promise.js:2970:14)
    at TaskQueue.executeNext_ (C:\HATG\ProtractorWorkspace\node_modules\selenium-webdriver\lib\promise.js:2953:27)
    at asyncRun (C:\HATG\ProtractorWorkspace\node_modules\selenium-webdriver\lib\promise.js:2860:25)
    at C:\HATG\ProtractorWorkspace\node_modules\selenium-webdriver\lib\promise.js:676:7
From: Task: Run it("should add a todo") in control flow
    at Object.<anonymous> (C:\HATG\ProtractorWorkspace\node_modules\jasminewd2\index.js:94:19)
    at attemptAsync (C:\HATG\ProtractorWorkspace\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:3945:24)
    at QueueRunner.run (C:\HATG\ProtractorWorkspace\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:3884:9)
    at C:\HATG\ProtractorWorkspace\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:3924:18
    at C:\HATG\ProtractorWorkspace\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:3848:9
    at Timeout.<anonymous> (C:\HATG\ProtractorWorkspace\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:3940:11)
    at ontimeout (timers.js:365:14)
    at tryOnTimeout (timers.js:237:5)
    at Timer.listOnTimeout (timers.js:207:5)
From asynchronous test: 
Error
    at Suite.<anonymous> (C:\HATG\protractor_07222017\test\example_spec.js:26:5)
    at addSpecsToSuite (C:\HATG\ProtractorWorkspace\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:994:25)
    at Env.describe (C:\HATG\ProtractorWorkspace\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:961:7)
    at describe (C:\HATG\ProtractorWorkspace\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4031:18)
    at Suite.<anonymous> (C:\HATG\protractor_07222017\test\example_spec.js:12:3)
    at addSpecsToSuite (C:\HATG\ProtractorWorkspace\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:994:25)
    at Env.describe (C:\HATG\ProtractorWorkspace\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:961:7)
    at describe (C:\HATG\ProtractorWorkspace\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4031:18)
    at Object.<anonymous> (C:\HATG\protractor_07222017\test\example_spec.js:1:1)]]>
   </failure>
   <failure type="exception" message="Failed: ENETUNREACH connect ENETUNREACH 172.20.10.13:63864"><![CDATA[Error: ENETUNREACH connect ENETUNREACH 172.20.10.13:63864
    at ClientRequest.<anonymous> (C:\HATG\ProtractorWorkspace\node_modules\selenium-webdriver\http\index.js:238:15)
    at emitOne (events.js:96:13)
    at ClientRequest.emit (events.js:189:7)
    at Socket.socketErrorListener (_http_client.js:358:9)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:189:7)
    at emitErrorNT (net.js:1280:8)
    at _combinedTickCallback (internal/process/next_tick.js:74:11)
    at process._tickCallback (internal/process/next_tick.js:98:9)
From: Task: WebDriver.navigate().to(data:text/html,<html></html>)
    at thenableWebDriverProxy.schedule (C:\HATG\ProtractorWorkspace\node_modules\selenium-webdriver\lib\webdriver.js:816:17)
    at Navigation.to (C:\HATG\ProtractorWorkspace\node_modules\selenium-webdriver\lib\webdriver.js:1140:25)
    at thenableWebDriverProxy.get (C:\HATG\ProtractorWorkspace\node_modules\selenium-webdriver\lib\webdriver.js:997:28)
    at driver.controlFlow.execute.then.then (C:\HATG\ProtractorWorkspace\node_modules\protractor\lib\browser.ts:885:30)
    at ManagedPromise.invokeCallback_ (C:\HATG\ProtractorWorkspace\node_modules\selenium-webdriver\lib\promise.js:1366:14)
    at TaskQueue.execute_ (C:\HATG\ProtractorWorkspace\node_modules\selenium-webdriver\lib\promise.js:2970:14)
    at TaskQueue.executeNext_ (C:\HATG\ProtractorWorkspace\node_modules\selenium-webdriver\lib\promise.js:2953:27)
    at asyncRun (C:\HATG\ProtractorWorkspace\node_modules\selenium-webdriver\lib\promise.js:2813:27)
    at C:\HATG\ProtractorWorkspace\node_modules\selenium-webdriver\lib\promise.js:676:7
    at process._tickCallback (internal/process/next_tick.js:103:7)
From: Task: Run beforeEach in control flow
    at Object.<anonymous> (C:\HATG\ProtractorWorkspace\node_modules\jasminewd2\index.js:94:19)
    at attemptAsync (C:\HATG\ProtractorWorkspace\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:3945:24)
    at QueueRunner.run (C:\HATG\ProtractorWorkspace\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:3884:9)
    at QueueRunner.execute (C:\HATG\ProtractorWorkspace\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:3872:10)
    at Spec.queueRunnerFactory (C:\HATG\ProtractorWorkspace\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:816:35)
    at Spec.execute (C:\HATG\ProtractorWorkspace\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:473:10)
    at Object.fn (C:\HATG\ProtractorWorkspace\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4975:37)
    at attemptAsync (C:\HATG\ProtractorWorkspace\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:3945:24)
    at QueueRunner.run (C:\HATG\ProtractorWorkspace\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:3884:9)
    at QueueRunner.execute (C:\HATG\ProtractorWorkspace\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:3872:10)
From asynchronous test: 
Error
    at Suite.<anonymous> (C:\HATG\protractor_07222017\test\example_spec.js:15:5)
    at addSpecsToSuite (C:\HATG\ProtractorWorkspace\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:994:25)
    at Env.describe (C:\HATG\ProtractorWorkspace\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:961:7)
    at describe (C:\HATG\ProtractorWorkspace\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4031:18)
    at Suite.<anonymous> (C:\HATG\protractor_07222017\test\example_spec.js:12:3)
    at addSpecsToSuite (C:\HATG\ProtractorWorkspace\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:994:25)
    at Env.describe (C:\HATG\ProtractorWorkspace\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:961:7)
    at describe (C:\HATG\ProtractorWorkspace\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4031:18)
    at Object.<anonymous> (C:\HATG\protractor_07222017\test\example_spec.js:1:1)]]>
   </failure>
   <failure type="exception" message="Failed: ENETUNREACH connect ENETUNREACH 172.20.10.13:63864"><![CDATA[Error: ENETUNREACH connect ENETUNREACH 172.20.10.13:63864
    at ClientRequest.<anonymous> (C:\HATG\ProtractorWorkspace\node_modules\selenium-webdriver\http\index.js:238:15)
    at emitOne (events.js:96:13)
    at ClientRequest.emit (events.js:189:7)
    at Socket.socketErrorListener (_http_client.js:358:9)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:189:7)
    at emitErrorNT (net.js:1280:8)
    at _combinedTickCallback (internal/process/next_tick.js:74:11)
    at process._tickCallback (internal/process/next_tick.js:98:9)
From: Task: WebDriver.navigate().to(data:text/html,<html></html>)
    at thenableWebDriverProxy.schedule (C:\HATG\ProtractorWorkspace\node_modules\selenium-webdriver\lib\webdriver.js:816:17)
    at Navigation.to (C:\HATG\ProtractorWorkspace\node_modules\selenium-webdriver\lib\webdriver.js:1140:25)
    at thenableWebDriverProxy.get (C:\HATG\ProtractorWorkspace\node_modules\selenium-webdriver\lib\webdriver.js:997:28)
    at driver.controlFlow.execute.then.then (C:\HATG\ProtractorWorkspace\node_modules\protractor\lib\browser.ts:885:30)
    at ManagedPromise.invokeCallback_ (C:\HATG\ProtractorWorkspace\node_modules\selenium-webdriver\lib\promise.js:1366:14)
    at TaskQueue.execute_ (C:\HATG\ProtractorWorkspace\node_modules\selenium-webdriver\lib\promise.js:2970:14)
    at TaskQueue.executeNext_ (C:\HATG\ProtractorWorkspace\node_modules\selenium-webdriver\lib\promise.js:2953:27)
    at asyncRun (C:\HATG\ProtractorWorkspace\node_modules\selenium-webdriver\lib\promise.js:2813:27)
    at C:\HATG\ProtractorWorkspace\node_modules\selenium-webdriver\lib\promise.js:676:7
    at process._tickCallback (internal/process/next_tick.js:103:7)
From: Task: Run beforeEach in control flow
    at Object.<anonymous> (C:\HATG\ProtractorWorkspace\node_modules\jasminewd2\index.js:94:19)
    at attemptAsync (C:\HATG\ProtractorWorkspace\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:3945:24)
    at QueueRunner.run (C:\HATG\ProtractorWorkspace\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:3884:9)
    at QueueRunner.execute (C:\HATG\ProtractorWorkspace\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:3872:10)
    at Spec.queueRunnerFactory (C:\HATG\ProtractorWorkspace\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:816:35)
    at Spec.execute (C:\HATG\ProtractorWorkspace\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:473:10)
    at Object.fn (C:\HATG\ProtractorWorkspace\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4975:37)
    at attemptAsync (C:\HATG\ProtractorWorkspace\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:3945:24)
    at QueueRunner.run (C:\HATG\ProtractorWorkspace\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:3884:9)
    at C:\HATG\ProtractorWorkspace\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:3924:18
From asynchronous test: 
Error
    at Suite.<anonymous> (C:\HATG\protractor_07222017\test\example_spec.js:15:5)
    at addSpecsToSuite (C:\HATG\ProtractorWorkspace\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:994:25)
    at Env.describe (C:\HATG\ProtractorWorkspace\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:961:7)
    at describe (C:\HATG\ProtractorWorkspace\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4031:18)
    at Suite.<anonymous> (C:\HATG\protractor_07222017\test\example_spec.js:12:3)
    at addSpecsToSuite (C:\HATG\ProtractorWorkspace\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:994:25)
    at Env.describe (C:\HATG\ProtractorWorkspace\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:961:7)
    at describe (C:\HATG\ProtractorWorkspace\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4031:18)
    at Object.<anonymous> (C:\HATG\protractor_07222017\test\example_spec.js:1:1)]]>
   </failure>
  </testcase>
 </testsuite>
</testsuites>